assignments/prototypes.js

```diff
@@ -33,6 +33,7 @@ function CharacterStats(param) {
 }
 CharacterStats.prototype=Object.create(GameObject.prototype);
 CharacterStats.prototype.takeDamage=function(){return `${this.name} took damage.`;}
+CharacterStats.prototype.destroy=function() {return `${this.name} was removed from the game`;};
```

Comment:
Thanks for mentioning this part. Great job on the task list and the frequent commits! 👍